OWML.Common/RegistrationCode.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Common
+{
+    public enum RegistrationCode
```

Comment:
Is this still in use?

Reply:
moved it into Input project and changed scope to internal